config/v1/types_console.go

```diff
@@ -35,3 +36,15 @@ type ConsoleList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Console `json:"items"`
 }
+
+type Auth struct {
+	// If authenticating with Request Header, OAuth or OpenID identity providers,
+	// an external URL is required to destroy single sign-on sessions.
+	LogoutRedirect string `json:"logoutRedirect"`
+}
+
+type Overrides struct {
+	// NodeSelector allows the console to be scheduled on nodes besides master nodes.
+	// +optional
+	NodeSelector metav1.LabelSelector `json:"nodeSelector,omitempty"`
```

Comment:
Thinking about where the `NodeSelector` belongs.  @derekwaynecarr mentioned in [this comment](https://github.com/openshift/api/pull/159#issuecomment-453263609) that it could just be done on the `Console` `cr.yaml`, perhaps we can remove it from here.

Reply:
Removed.